### PR TITLE
Change of page structure and added content to match original page

### PR DESCRIPTION
**Pull request by edemagbenyo**
The following have been made:
Structure of the entire page
Made page responsive for various breakpoints
Added content to the page to bring out its beauty

What we learnt:
Media Query
Grid responsive design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,606 @@
...
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+	<link rel="stylesheet" type="text/css" href="resources/css/main.css">
+</head>
+
+<body>
+	<header>
+		<div class="nav-container">
+			<div class="logo"></div>
```
Comment (bolah2009) [inline index.html]: This logo is supposed to appear for smaller screens, right?
It seems not to be showing.

-  [ ] TWN logo appears at the leftmost part.


| Current | Source |
| :-----: | :----- |
| tnw-current-small | tnw-small-screen-source |

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Good start team :smile: 

### [The clone](https://rawcdn.githack.com/collinsugwu/the-next-web/7f9bdb95937781e96a48d7fa2322114176f699c6/index.html) looks good to me and responsive at major breakpoints. :100: 

Let's fix the issue in the comment below. Also, let's edit the PR name and description to a more descriptive one. :blush: 

#### :recycle:  Make these changes and submit another code review request. :+1: 

#### Feel free to reach me on slack if you having issues implementing these features. :+1: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: index.html [hunk 2]
```diff
@@ -13,7 +13,9 @@
 <body>
 	<header>
 		<div class="nav-container">
-			<div class="logo"></div>
+			<div class="logo">
+				<img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="tnw logo">
+			</div>
```
Comment (samgaco) [inline index.html:18]: :100:

**Review by samgaco — APPROVED** [review]
Hi Team :high_brightness: 

All the changes have been met, therefore the project is approved :white_check_mark:  You may merge to the master branch and submit the project completion form.

Keep up the good work moving forward :muscle: 

-------------------
* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**